SquAPI.lua

```diff
@@ -73,1100 +68,1238 @@ squapi.autoFunctionUpdates = true
-                self.taurBody:setRot(80, 0, 0)
-                if self.backLegs then
-                    self.backLegs:setRot(-50, 0, 0)
-                end
-                if self.frontLegs then
-                    self.frontLegs:setRot(-50, 0, 0)
-                end
-            else	
-                self.taurBody:setRot(self.taur.pos, 0, 0)
-                if self.backLegs then
-                    self.backLegs:setRot(self.taur.pos*3, 0, 0)
-                end
-                if self.frontLegs then
-                    self.frontLegs:setRot(-self.taur.pos*3, 0, 0)
-                end
-            end
-        end
-	end
+  ---Sets if this taur is enabled
+  ---@param bool boolean
+  function _self:setEnabled(bool)
+    assert(type(bool) == "boolean",
+      "§4setEnabled must be set to a boolean.§c")
+    _self.enabled = bool
+  end
 
-    table.insert(squapi.taurs, self)
-    return self
-end
+  -- UPDATES -------------------------------------------------------------------------
 
+  ---Run tick function on taur
+  function _self:tick()
+    if _self.enabled then
+      _self.target = math.min(math.max(-30, squassets.verticalVel() * 40), 45)
+    end
+  end
 
+  ---Run render function on taur
+  ---@param dt number Tick delta
+  function _self:render(dt, _)
+    if _self.enabled then
+      _self.taur:berp(_self.target, dt / 2)
+      local pose = player:getPose()
+
+      if pose == "FALL_FLYING" or pose == "SWIMMING" or (player:isClimbing() and not player:isOnGround()) or player:riptideSpinning() then
+        _self.taurBody:setRot(80, 0, 0)
+        if _self.backLegs then
+          _self.backLegs:setRot(-50, 0, 0)
+        end
+        if _self.frontLegs then
+          _self.frontLegs:setRot(-50, 0, 0)
+        end
+      else
+        _self.taurBody:setRot(_self.taur.pos, 0, 0)
+        if _self.backLegs then
+          _self.backLegs:setRot(_self.taur.pos * 3, 0, 0)
+        end
+        if _self.frontLegs then
+          _self.frontLegs:setRot(-_self.taur.pos * 3, 0, 0)
+        end
+      end
+    end
+  end
 
--- CUSTOM FIRST PERSON HAND
---!!Make sure the setting for modifying first person hands is enabled in the Figura settings for this to work properly!!
--- guide:(note if it has a * that means you can leave it blank/nil to use reccomended settings)
--- element:	the actual hand element to change
--- *x: 		        the x change
--- *y: 		        the y change
--- *z: 		        the z change
--- *scale:          this will multiply the size of the element by this size
--- *onlyVisibleInFP:if true, this will make the element invisible when not in first person
+  table.insert(squapi.taurs, _self)
+  return _self
+end
 
+---Contains all registered first person hands
+---@type SquAPI.FPHand[]
 squapi.FPHands = {}
 squapi.FPHand = {}
 squapi.FPHand.__index = squapi.FPHand
+
+---CUSTOM FIRST PERSON HAND<br>**!!Make sure the setting for modifying first person hands is enabled in the Figura settings for this to work properly!!**
```

Comment:
Yet another `<br>` tag.
```suggestion
---CUSTOM FIRST PERSON HAND  
---**!!Make sure the setting for modifying first person hands is enabled in the Figura settings for this to work properly!!**
```